packages/reports/addon/templates/components/dimension-selector.hbs

```diff
@@ -1,38 +1,58 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#navi-list-selector
-  items=listItems
-  searchField="longName"
-  selected=selectedColumnsAndFilters
-  title="Dimensions"
+<NaviListSelector
+  @items={{this.listItems}}
+  @searchField="longName"
+  @selected={{this.selectedColumnsAndFilters}}
+  @title="Dimensions"
   as | items areItemsFiltered |
-}}
-  {{#grouped-list
-      items=items
-      shouldOpenAllGroups=areItemsFiltered
-      groupByField="category"
-      sortByField=(unless areItemsFiltered "longName")
-      as | item |
-  }}
-    <label class="grouped-list__item-label">
-      <input class="checkbox-selector__checkbox" type="checkbox" onchange={{action "itemClicked" item}} checked={{get itemsChecked item.name}}>
-      {{item.longName}}
-    </label>
-    {{#unless (eq item.category "Time Grain")}}
-      {{#navi-icon "question-circle-o" class="grouped-list__item-info"}}
-        {{#ember-tooltip side="right" popperContainer="body" effect="none"}}
-          {{#if item.extended.isPending}}
-            {{navi-loading-message}}
-          {{else}}
-            {{or item.extended.content.description "No description available"}}
-          {{/if}}
-        {{/ember-tooltip}}
-      {{/navi-icon}}
+>
+  <GroupedList
+    @items={{items}}
+    @shouldOpenAllGroups={{areItemsFiltered}}
+    @groupByField="category"
+    @sortByField={{unless areItemsFiltered "longName"}}
+    as | item |
+  >
+    {{#if (eq item.category "Time Grain")}}
+      <div class="grouped-list__item-checkbox-container">
+        <input
+          type="checkbox"
+          class="grouped-list__item-checkbox"
+          id={{concat "grouped-list-checkbox-" (dasherize item.longName)}}
+          onchange={{action "itemClicked" item}}
+          checked={{get itemsChecked item.name}}>
```

Comment:
Can we move the `/>` to the next line? I was searching for where this input ends. For some tags, we do move them to the next line, and for others we don't.